hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/BatchAPI.java

```diff
@@ -71,4 +78,75 @@ public BatchAPI() {
             batchWriteThreads.decrementAndGet();
         }
     }
+
+    // TODO: Design more flexible for Element?
```

Comment:
seems unused TODO

Reply:
I was wondering if I could make it easier to merge `JsonElement` & `Element` by extends or implements a class, will del it